package.json

```diff
@@ -28,6 +28,9 @@
     "start": "node srv/start.js"
```

Comment:
Add the lint command here (same as search) So people with no editor (vim for instance) can lint by running `npm run lint`